pkgs/applications/networking/instant-messengers/psi-plus/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub,
+  which, pkgconfig,
+  qt4, libidn, qca2, libXScrnSaver, enchant,
+}:
+
+stdenv.mkDerivation rec {
+  name = "psi-plus-${version}";
+  version = "0.16.572.639";
+
+  src = fetchFromGitHub {
+    owner = "psi-plus";
+    repo = "psi-plus-snapshots";
+    rev = "${version}";
+    sha256 = "0nlqca5z22wl290scv8lk8hlm188rj2h63lf662ayzkqyjwq7zgm";
+  };
+
+  resources = fetchFromGitHub {
+    owner = "psi-plus";
+    repo = "resources";
+    rev = "8f5038380e1be884b04b5a1ad3cc3385e793f668";
+    sha256 = "1b8a2aixg966fzjwp9hz51rc31imyvpx014mp2fsm47k8na4470d";
+  };
+
+  postUnpack = ''
+    cp -a "${resources}/iconsets" "$sourceRoot"
+  '';
+
+  buildInputs = [
+    which pkgconfig
+    qt4 libidn qca2 libXScrnSaver enchant
+  ];
+
+  NIX_CFLAGS_COMPILE="-I${qca2}/include/QtCrypto";
+
+  NIX_LDFLAGS="-lqca";
+
+  enableParallelBuilding = true;
+
+  meta = {
+    description = "Psi+, an XMPP (Jabber) client";
```

Comment:
Thank you for the review, I updated the pull request. `which` is needed, otherwise `configure` fails with `You don't seem to have 'make' or 'gmake' in your PATH.`.